src/ppx/scene/scene_scene.cpp

```diff
@@ -0,0 +1,126 @@
+#include "ppx/scene/scene_scene.h"
+
+namespace ppx {
+namespace scene {
+
+Scene::Scene(std::unique_ptr<scene::ResourceManager>&& resourceManager)
+    : mResourceManager(std::move(resourceManager))
+{
+}
+
+Scene::~Scene()
+{
+    mResourceManager->DestroyAll();
```

Comment:
This is probably not necessary -> the manager is deleted in this dtor, which will clear the vectors and delete resources.

Reply:
Removed.